lib/ts/recipe/thirdparty/components/themes/styles.ts

```diff
@@ -17,14 +17,7 @@ import chroma from "chroma-js";
 import { getButtonStyle, getDefaultStyles, getMergedStyles } from "../../../../styles/styles";
 import { NormalisedPalette, NormalisedDefaultStyles } from "../../../../types";
 
-const providerColors = {
-    google: "#ea3721",
-    github: "#000",
-    facebook: "#274483",
-    twitter: "#008dd1",
-    apple: "#07093c",
-    custom: "#FFF",
```

Comment:
please redo how custom provider's default UI button shows -> the alignment of the text in the button is a bit off